scipy/stats/_qmc.py

```diff
@@ -1083,7 +1100,7 @@ class LatinHypercube(QMCEngine):
     distinct rows occur the same number of times. The elements of :math:`A`
     are in the set :math:`\{0, 1, ..., p-1\}`, also called symbols.
     The constraint that :math:`p` must be a prime number is to allow modular
-    arithmetic.
+    arithmetic. The strength add some symettry in sub-projections to a sample.
```

Comment:
"symmetry"
I don't immediately understand what this means or what the implications are. Expand, perhaps.

Reply:
Let me know if this is better. I would like to be cautious with the wording as I am not 100% if the symmetry is perfect and how it behaves.

Reply:
I think it helps understand the math gibberish. Good or bad is always a matter of perspective. It will cover better the space, but the patterns can be seen as a default (e.g. if you observe periodic phenomena and are not lucky, you can miss that with periodic/symmetric samples). I edited.